app/controllers/api/v1x2/stageaction/show.html.erb

```diff
@@ -4,66 +4,44 @@
-                        <div class="pf-l-split__item pf-m-fill"><%= v %></div>
-                      </div>
-                    </div>
-                  <% end %>
-                </div>
-              </div>
-            </div>
+        </div>
+        <p>
+        <div class="install-title">Comment</div>
+        <%= form_tag({:action => 'update', :controller => 'stageaction'}, :method => :patch) do %>
+        <div class="pf-l-grid pf-m-gutter">
+          <%= text_area_tag(:message, '', :class => "pf-l-grid__item pf-m-12-col-on-xs pf-m-6-col-on-sm pf-m-6-col-on-lg pf-m-6-col-on-xl", :style => "height: 100px") %>
+        </div>
+        <br>
+        <%= hidden_field_tag('approver', Base64.encode64(@approver)) %>
+        <div>
+            <%= button_tag 'Approve', :class => "pf-c-button pf-m-primary", :type => "submit", :name => :commit, :value => "Approve" %>
+
+            <%= button_tag 'Deny', :class => "pf-c-button pf-m-danger", :type => "submit", :name => :commit, :value => "Deny" %>
+
+           <%= button_tag 'Comment', :class => "pf-c-button pf-m-secondary", :type => "submit", :name => :commit, :value => "Memo" %>
           </div>
-          <p>
-          <div class="install-title">Reason/Memo</div>
-          <%= form_tag({:action => 'update', :controller => 'stageaction'}, :method => :patch) do %>
-          <div class="pf-l-grid pf-m-gutter">
-            <%= text_area_tag(:message, '', :class => "pf-l-grid__item pf-m-12-col-on-xs pf-m-6-col-on-sm pf-m-6-col-on-lg pf-m-6-col-on-xl", :style => "height: 100px") %>
-          </div>
-          <br>
-          <%= hidden_field_tag('approver', Base64.encode64(@approver)) %>
-          <div class="pf-l-grid pf-m-gutter">
-            <div class="pf-l-grid__item pf-m-12-col-on-xs pf-m-2-col-on-sm pf-m-2-col-on-lg pf-m-2-col-on-xl">
-             <%= button_tag 'Memo', :class => "pf-c-button pf-m-secondary pf-m-block", :type => "submit", :name => :commit, :value => "Memo" %>
-            </div>
-            <div class="pf-l-grid__item pf-m-12-col-on-xs pf-m-2-col-on-sm pf-m-2-col-on-lg pf-m-2-col-on-xl">
-              <%= button_tag 'Approve', :class => "pf-c-button pf-m-primary pf-m-block", :type => "submit", :name => :commit, :value => "Approve" %>
-            </div>
-            <div class="pf-l-grid__item pf-m-12-col-on-xs pf-m-2-col-on-sm pf-m-2-col-on-lg pf-m-2-col-on-xl">
-              <%= button_tag 'Deny', :class => "pf-c-button pf-m-danger pf-m-block", :type => "submit", :name => :commit, :value => "Deny" %>
-            </div>
-          <% end %>
-          <script>
-              function showDetails() {
-                  var x = document.getElementById("details");
-                  if (x.style.display === "none") {
-                    x.style.display = "block";
-                  } else {
-                    x.style.display = "none";
-                  }
-              }
-          </script>
-        </section>
+        <% end %>
```

Comment:
What is the the `end` of? Maybe I missed it but I don't see any opening `do` or `if/else` or something like that.